README.md

```diff
@@ -5,6 +5,7 @@
 Build state: 
 ![GitHub Actions build state](https://github.com/pi4j/pi4j-v2/workflows/Continious%20Integration/badge.svg)
 
+[![Chat on Slack](https://img.shields.io/badge/Chat-on%20Slack-blue)](https://join.slack.com/t/pi4j/shared_invite/zt-1ttqt8wgj-E6t69qaLrNuCMPLiYnBCsg)
```

Comment:
when I updated my local repo prior to the latest push it put this change as a new edit, even though this is the code in the remote repo.   Not sure why/how but assume a result of the git commands I used.